packages/components/src/pages/VsCodeExtension.vue

```diff
@@ -572,8 +580,28 @@ export default {
         });
       }
 
+      let eventIds = [];
+      if (this.traceFilterValue) {
+        const filterValue = this.traceFilterValue
+          .replaceAll(/\s/g, '')
```

Comment:
Prefer `replace(/\s+/g, '')` over replaceAll here - it has higher browser support coverage.